ExampleMod/Content/Items/Consumables/MinionBossSummonItem.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Terraria;
+using Terraria.Audio;
+using Terraria.ID;
+using Terraria.GameContent.Creative;
+using Terraria.ModLoader;
+using ExampleMod.Content.NPCs.MinionBoss;
+
+namespace ExampleMod.Content.Items.Consumables
+{
+	//This is the item used to summon a boss, in this case the modded Minion Boss from Example Mod. For vanilla boss summons, see PlanteraItem.cs
+	public class MinionBossSummonItem : ModItem
+	{
```

Comment:
May wanna remove that plantera summoning item we already had.

Reply:
I added a few comments to `MinionBossSummonItem`, making the plantera summoning item obsolete. Can be removed when this is merged.